extension.js

```diff
@@ -356,6 +368,25 @@ function update_height_setting(win) {
     settings.set_double('window-height', Math.min(1.0, current_height));
 }
 
+function toggle_window_maximize() {
```

Comment:
Maybe yes. 

Tbh, I wanted to change `unset_window_maximize` but couldn't come up with anything. Maybe `unset_window_maximize` / `set_window_maximized` is a good compromise? Otherwise don't know what to use.

Reply:
`disable_window_maximize_setting` ?